sections/05_treat.md

```diff
@@ -11,6 +11,148 @@ other interventions?  As an example, [@doi:10.1016/j.jalz.2015.01.010]
 predicts individuals who are most likely to decline during a clinical trial
 and benefit from the treatment.*
 
+There has been sustained interest in applying deep learning to clinical
+decision making for over two decades. In 1996, Tu
+[@doi:10.1016/S0895-4356(96)00002-9] compared the effectiveness of artificial
+neural networks and logistic regression, questioning whether deep learning
+would replace traditional statistical methods for predicting medical
+outcomes. He posited that while neural networks have several advantages in
+representational power, the difficulties in interpretation may limit clinical
+applications. In 2007, Lisboa and Taktak [@doi:10.1016/j.neunet.2005.10.007]
+examined the use of artificial neural networks in medical journals, concluding
+that neural networks provided an increase in benefit to healthcare provision
+in 21 of 27 studies.
+
+While significant progress has been made in developing deep learning methods
+for diagnosis, it is not clear that these methods have yet transformed
+clinical decision making. The difficulty in applying deep learning to clinical
+decision making represents a challenge common to many deep learning
+applications: it is much easier to predict an outcome than to suggest an
+action to change the outcome. Several attempts at recasting the clinical
+decision making problem into a prediction problem (i.e. prediction of which
+treatment will most improve the patient's health) have accurately predicted
+prescription habits, but technical and medical challenges remain for clinical
+adoption. In particular, remaining challenges include actionable
+interpretability of deep learning models, fitting deep models to limited and
+heterogeneous data, and integrating complex predictive models into a dynamic
+clinical environment.
+
+#### Applications
+
+##### Trajectory Prediction for Treatment
+
+A common application for deep learning techniques in this domain is to
+leverage the temporal structure of healthcare records. As previously
+discussed, many studies [@arxiv:1606.04130 @arxiv:1606.01865 @doi:10.1109/ACCESS.2016.2618775 @arxiv:1510.07641]
+have used deep recurrent networks to categorize patients but most stop short
+of suggesting clinical decisions. Nemati et al [@doi:10.1109/EMBC.2016.7591355]
+used deep reinforcement learning to optimize a heparin dosing policy for
+intensive care patients. However, because the ideal dosing policy is unknown,
+the model's predictions must be evaluated on counter-factual data. This
+represents a common challenge when bridging the gap between research and
+clinical practice: because the ground-truth is unknown, researchers struggle
+to evaluate model predictions in the absence of interventional data, but
+clinical application is unlikely until the model has been shown to be effective
+. The impressive applications of deep reinforcement learning to other domains
+(@doi:10.1038/nature16961) have relied on knowledge of the underlying
```

Comment:
Add the square brackets instead the parentheses so the parser recognizes the reference: `[(@doi:10.1038/nature16961)]`